source/Halibut.Tests/UsageFixture.cs

```diff
@@ -125,7 +125,7 @@ public async Task StreamsCanBeSent(ClientAndServiceTestCase clientAndServiceTest
         }
 
         [Test]
-        [LatestAndPreviousClientAndServiceVersionsTestCases(testNetworkConditions: false)]
+        [LatestAndPreviousClientAndServiceVersionsTestCases(testWebSocket:true, testNetworkConditions: false)]
```

Comment:
My understanding of the outcome of standup is we would be doing very little tests on websockets since they are so unreliable.

@nathanwoctopusdeploy ?

Reply:
Oh i see so we should keep marking the tests as run on websockets but don't actually run them. Thanks will continue to do that on my changes as well.

Reply:
Yeah - if we have existing tests that already pass w/ websockets (which we can verify locally, by enabling websockets in `ServiceConnectionTypes` and testing not net48), then we can mark the websocket columns as 'done' for those tests.

But I don't think we want to spend time writing new tests, or fixing old tests, to get them to pass w/ websockets.